Documentation/ceph-osd-mgmt.md

```diff
@@ -63,23 +60,51 @@ through the removal process:
 If all the PGs are `active+clean` and there are no warnings about being low on space, this means the data is fully replicated
 and it is safe to proceed. If an OSD is failing, the PGs will not be perfectly clean and you will need to proceed anyway.
 
-### From the Toolbox
+### Prevent the rook operator to re-create the OSD pod
+
+#### Host-based cluster
+
+Update your CephCluster CR. Depending on your CR settings, you may need to remove the device from the list or update the device filter.
+If you are using `useAllDevices: true`, no change to the CR is necessary.
+
+#### PVC-based cluster
+
+1. Shrink the number of OSDs in the `storageClassDeviceSet` in the CephCluster CR.
+   - `kubectl -n rook-ceph edit cephcluster rook-ceph`
+   - Reduce the `count` of the OSDs to the desired number. Rook will not take any action to automatically remove the extra OSD(s), but will effectively stop managing the orphaned OSD.
+2. Identify the orphaned PVC that belongs to the orphaned OSD.
+   - The orphaned PVC will have the highest index among the PVCs for the device set.
+   - `kubectl -n rook-ceph get pvc -l ceph.rook.io/DeviceSet=<deviceSet>`
+   - For example if the device set is named `set1` and the `count` was reduced from `3` to `2`, the orphaned PVC would have the index `2` and might be named `set1-2-data-vbwcf`
+3. Identify the orphaned OSD.
+   - The OSD assigned to the PVC can be found in the labels on the PVC
+   - `kubectl -n rook-ceph get pod -l ceph.rook.io/pvc=<orphaned-pvc> -o yaml | grep ceph-osd-id`
+   - For example, this might return: `ceph-osd-id: "0"`
+
+### Confirm the OSD is down
+
+If you want to remove a unhealthy OSD, the osd pod may be in an error state such as `CrashLoopBackoff` or the `ceph` commands
+in the toolbox may show which OSD is `down`. If you want to remove a healthy OSD, you should run
+`kubectl -n rook-ceph scale deployment rook-ceph-osd-<ID> --replicas=0` and `ceph osd down osd.<ID>` from the toolbox.
 
-1. Determine the OSD ID for the OSD to be removed. The osd pod may be in an error state such as `CrashLoopBackoff` or the `ceph` commands
-in the toolbox may show which OSD is `down`. If you want to remove a healthy OSD, you should run `kubectl -n rook-ceph scale deployment rook-ceph-osd-<ID> --replicas=0` beforehand.
-2. Mark the OSD as `out` if not already marked as such by Ceph. This signals Ceph to start moving (backfilling) the data that was on that OSD to another OSD.
+### Purge the OSD from Ceph cluster
+
+OSD removal can be automated with the example found in the [rook-ceph-purge-osd job](/cluster/examples/kubernetes/ceph/osd-purge.yaml).
+You can go to [Delete the underlying data][#delete-the-upderlying-data] section after running this job. If you want to remove OSD by hand,
```

Comment:
I think it's ok for them to find the section later for deleting underlying data.
```suggestion
If you want to remove OSD by hand,
```

Reply:
Fixed, thanks.